ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -221,12 +221,20 @@ define('setup_manage_taxonomies', ['reactaddons', 'lodash', 'jquery', 'utils'],
 
       var checkboxes = _.map(this.props.learningResourceTypes, function(type) {
         var checked = _.includes(thiz.state.learningResourceTypes, type);
-        return <li key={type}><label><input type="checkbox"
-                      value={type}
-                      checked={checked}
-                      onChange={thiz.updateLearningResourceType} />
-          {type}
-        </label></li>;
+        return (
+            <li className="icheck-vocab-lr-types" key={type}>
```

Comment:
Since you are not using icheck , maybe rename the class name to something else to avoid confusion.